scripts/BumpVersion.swift

```diff
@@ -0,0 +1,169 @@
+import Foundation
+
+let greenColor = "\u{001B}[0;32m"
+let resetColor = "\u{001B}[0;0m"
+let redColor = "\u{001B}[0;31m"
+let yellowColor = "\u{001B}[33m"
+
+guard CommandLine.arguments.count == 2 else {
+    print("\(redColor)Please provide a version in the format `major.minor.patch`\(resetColor)")
+    exit(1)
+}
+
+func isVersionBumped(_ currentVersion: String, _ newVersion: String) -> Bool {
```

Comment:
Rather than implement this manually, we use a package called `semver` in React Native macOS to help. This is where we use it: https://github.com/microsoft/react-native-macos/blob/main/.ado/get-next-semver-version.js

A quick google says similar stuff may be available for Swift: https://swiftpackageindex.com/sersoft-gmbh/semver

Reply:
This seems pretty cool actually! Though I'm not not sure if it's worth adding this as a package in fluent

Reply:
Yeah that makes sense. Thought I'd share anyway

Reply:
Totally, thanks for doing so!